squiggy/models/course.py

```diff
@@ -135,7 +135,7 @@ def get_advanced_asset_search_options(
                 JOIN courses c ON c.id = u.course_id
                 WHERE
                     u.course_id = :course_id
-                    AND canvas_enrollment_state = 'active'
+                    AND canvas_enrollment_state in ('active', 'invited')
```

Comment:
@mlincoln - Not a merge blocker but...  I typically capitalize the SQL reserved words – in this case, "IN".

Reply:
I'll amend, and submit a separate fix for dev.